infrasim/model/tasks/compute.py

```diff
@@ -224,13 +223,9 @@ def init(self):
             imc_obj = CPCIIMC(imc_element)
             self.__element_list.append(imc_obj)
 
-        for pcu_cr0_element in self.__compute.get("pcu_cr0", []):
-            pcu_cr0_obj = CPCIPCU_CR0(pcu_cr0_element)
-            self.__element_list.append(pcu_cr0_obj)
-
-        for pcu_cr5_element in self.__compute.get("pcu_cr5", []):
-            pcu_cr5_obj = CPCIPCU_CR5(pcu_cr5_element)
-            self.__element_list.append(pcu_cr5_obj)
+        for pcu_element in self.__compute.get("pcu", []):
```

Comment:
suggest to move this code snippet behind 
```python
self.__element_list.append(backend_network_obj)
```
otherwise, it will be failed if this device need to be re-position by pcie_topology module due to behindhand identifier of bus_id.

Reply:
Yes, this is a potential issue. Thanks.